app/views/me/notices/index.html.haml

```diff
@@ -1,12 +1,24 @@
-.h2.first=:notices.t
+%div
+  %span.h2.first=:notices.t
+  - if @notices.total_pages < 2 && @notices.any?
+    %span.h2.dissmiss-all-notices-button.dismiss-all-wrapper.pull-right
```

Comment:
you probably know this, but you can update this same PR by doing a `push --force`. e.g.

    git checkout hide_all_notifications_link
    git push --force aliaksandrb hide_all_notifications_link

where `aliaksandrb` is the name of the remote you have added for your github account.